packages/minimongo/modify.js

```diff
@@ -185,20 +185,20 @@ var MODIFIERS = {
   $currentDate: function (target, field, arg) {
     if (typeof arg === "object" && arg.hasOwnProperty("$type")) {
        if (arg.$type !== "date") {
-         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers");
+         throw MinimongoError("Minimongo does currently only support the date type in $currentDate modifiers: " + field);
```

Comment:
Can we make the `MinimongoError` constructor take a second (optional) parameter, instead of concatenating the strings together here?